### PR TITLE
User Select Demo Cleanup

### PR DESCRIPTION
**Pull request by ststimac**
New PR for the user select demo. This should be free of other demo conflicts & only be the user select files. 

@melanierichards

### CODE REVIEW

File: svgradientbackgroundmaker/index.html [hunk 1]
```diff
@@ -16,7 +16,7 @@
 </head>
 
 <body>
-	<article class="container">
+	<article class="container">
```
Comment (dstorey) [inline svgradientbackgroundmaker/index.html]: White space needs removed

Comment (dstorey) [reply svgradientbackgroundmaker/index.html]: ...and .DS-Storey below. Wont let me comment on that

File: userselect/images/avatar_1.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 34 34"><style>.st0{clip-path:url(#SVGID_2_);} .st1{fill:#D2D2D2;} .st2{fill:#4E4E4E;}</style><defs><path id="SVGID_1_" d="M0-.1h34v34.2H0z"/></defs><clipPath id="SVGID_2_"><use xlink:href="#SVGID_1_" overflow="visible"/></clipPath><g class="st0"><path class="st1" d="M-4.1-3h41.6v41.6H-4.1z"/><circle class="st2" cx="16.7" cy="12.4" r="7.1"/><path class="st2" d="M16.7 56.7c-7.5 0-13.6-6.1-13.6-13.6v-9c0-7.5 6.1-13.6 13.6-13.6s13.6 6.1 13.6 13.6v9c0 7.5-6.1 13.6-13.6 13.6z"/></g></svg>
```
Comment (dstorey) [inline userselect/images/avatar_1.svg]: Can you run this and the other svg through an optimiser to remove the unneeded id/class , etc? Lower case hex codes too (our linter doesn't pick them up in SVG :()

File: userselect/index.html [hunk 3]
```diff
@@ -7,175 +7,185 @@
...
-					<option value="normal_off">Turn off selection except in editable content</option>
-					<option value="all_off">Turn off selection everywhere</option>
-					<option value="blog_only">Select Blog Post Content only</option>
-					<option value="comments_only">Select Comments only</option>
-					<option value="blog_beyond">Selection Starts in Blog Post or Comments but can extend</option>
-				</select>
-			</div>
-			<div id="code"> <pre id="normal_off">
+</header>
+<section role="region "class="container section">
```
Comment (dstorey) [inline userselect/index.html]: role attribute isn't needed anymore

File: userselect/index.html [hunk 4]
```diff
@@ -7,175 +7,185 @@
...
+<div class="userselect-container clear">
+	<div id="controls">
+		<select id="sel" class="controls-select">
```
Comment (dstorey) [inline userselect/index.html:36]: looks like this select needs a label

File: userselect/index.html [hunk 5]
```diff
@@ -7,175 +7,185 @@
...
+		</select>
+	<div id="code">
+<pre id="normal_off">
 <code>#blog {
-	-ms-user-select:none;
-	-webkit-user-select:none;
-	-moz-user-select:-moz-none;
-	} </code></pre>
-            <pre id="all_off">
+-ms-user-select:none;
```
Comment (dstorey) [inline userselect/index.html]: spaces between : and value

File: userselect/index.html [hunk 6]
```diff
@@ -7,175 +7,185 @@
...
-				<span>&#160; &nbsp;</span>
-
-				<div style="color: #0066aa; font-weight: bold; float: left;"><a href="#">IEBlog</a>
+	</div>
+	<div id="blog">
+		<div id="sample-content">
+			<img src="images/user-select.svg" alt="IEBlog The Internet Explorer Weblog" />
+		</div>
+		<p class="breadcrumb-list">
+			<a href="#">MSDN Blogs</a> \
```
Comment (dstorey) [inline userselect/index.html]: we may as well add the real href values here to avoid any potential a11y flags here

File: userselect/index.html [hunk 7]
```diff
@@ -7,175 +7,185 @@
...
+			<a href="#">IEBlog</a> \
+			HTML5 History in IE10
+		</p>
+
+		<p class="title">HTML5 History in IE10</p>
+		<p class="date"><a href="#">ieblog</a> 31 Oct 2011 3:42 PM</p>
```
Comment (dstorey) [inline userselect/index.html]: Same comment as above

File: userselect/index.html [hunk 8]
```diff
@@ -7,175 +7,185 @@
...
+			the Windows Developer Preview eliminates the compromise by adding support for HTML5 History. The pushState,
+			replaceState, and popstate APIs provide fine-grained control over the behavior of the back button and
+			the URL presented to the user for dynamic content. Together these APIs help you improve the performance of
+			your site without sacrificing usability.</p>
+		<p>...</p>
+
+		<p class="subtitle">Comments</p>
+		<div class="comments">
+			<div class="comment">
+				<img src="images/avatar_1.svg" class="avatar" alt="avatar" role="presentation" />
```
Comment (dstorey) [inline userselect/index.html]: if this has role presentation then the alt attribute is not needed, as that removes from a11y tree. Role should be first attribute (in our style guide) as it should be close to the semantic that it is replacing (the start tag)

File: userselect/index.html [hunk 9]
```diff
@@ -7,175 +7,185 @@
...
+				<div class="comment__content">
+					<p class="comment--user">Paul</p>
+					<p>Excellent and much needed feature!</p>
+				</div>
+			</div>
+			<div class="comment--reply clear">
+				<img src="images/avatar_1.svg" class="avatar" alt="avatar" role="presentation" />
```
Comment (dstorey) [inline userselect/index.html]: Same comment as above

File: userselect/index.html [hunk 10]
```diff
@@ -7,175 +7,185 @@
...
-				<p>Thank you!!!</p></div>
-			<h3>Leave a Comment</h3>
-			<label>Name</label><br/>
-			<input type="text" style="width:400px;"><br/>
-			<label>Comment</label><br/>
-			<textarea style="width:400px;height:100px;"></textarea>
 		</div>
+		<p class="subtitle">Leave a Comment</p>
+		<label>Name</label><br/>
+		<input type="text"><br/><br/>
```
Comment (dstorey) [inline userselect/index.html]: needs label associated with input (id/for attributes)

File: userselect/index.html [hunk 11]
```diff
@@ -7,175 +7,185 @@
...
-			property. Both <a href="https://developer.mozilla.org/en/CSS/-moz-user-select ">Mozilla</a> and <a
-				href="http://developer.apple.com/library/safari/#documentation/AppleApplications/Reference/SafariCSSRef/Articles/StandardCSSProperties.html#//apple_ref/css/property/-webkit-user-select ">Webkit</a>
-			support their own prefixed versions of this property. There are minor differences between the three
-			implementations so be sure to test your application across browsers.</p>
-		<br/>
-	</div>
-</div>
-<script src="scripts/demo.js"></script>
+</section>
+<section role="region" class="container section">
```
Comment (dstorey) [inline userselect/index.html]: role is not needed any longer here

**Review by dstorey — CHANGES_REQUESTED** [review]

File: userselect/images/avatar_1.svg (hunk 2 above)
Comment (ststimac) [reply userselect/images/avatar_1.svg]: I ran them through one yesterday. I'll try again.

File: userselect/index.html (hunk 6 above)
Comment (ststimac) [reply userselect/index.html]: I'd rather just delete the links so people stay on the demo page.

**Review by dstorey — APPROVED** [review]

**Comment by melanierichards** [comment]
Looks good!

**Comment by ststimac** [comment]
Hooray! ?

Sent using OWA for iPhone
________________________________

**Pull request merged**